style.css

```diff
@@ -0,0 +1,27 @@
+h1{
+    color:blue
+}
+.bgimage {
+    height:100vh;
+    background: url('https://static.wikia.nocookie.net/marveldatabase/images/2/20/Iron_Man_Armor_Model_3_from_All-New_Iron_Manual_Vol_1_1_001.jpg/revision/latest/scale-to-width-down/250?cb=20170122055210');
```

Comment:
Unfortunately, this image doesn't seem to load (this is actually a security feature; the Marvel Wiki doesn't want people embedding the image on your website). 

Maybe try one of the placeholder images instead?